src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type MoviesListProps = {
+  movies: Movie[]
+};
+
+export const MoviesList: React.FC<MoviesListProps> = ({ movies }) => (
+  <div className="movies-list">
```

Comment:
```suggestion
  <div className="movies">
```
I guess we don't have `movies-list` class in our css files

Reply:
Thank you, I could not find where I made a mistake that it looks so strange.